server/controllers/booking/booking.controller.js

```diff
@@ -2,19 +2,26 @@ import Booking from "../../models/booking";
 import car from "../../models/car";
 import User from "../../models/user";
 
-
 import moment from "moment";
 
-const codeGenerator = require('../../util/code.generator');
+const codeGenerator = require("../../util/code.generator");
 
 const createBooking = function(req, res) {
   const startAt = req.body.startAt;
   const endAt = req.body.endAt;
   const carid = req.body.car;
   const userid = req.body.userid;
 
-  if(!userid || !carid)
-  {
+  if (
```

Comment:
Both of these checks are the same, do you need 2 of them?

Reply:
No, also corrected